### DISCUSSION TITLE
Issues Deploying my Python FastAPI

### DISCUSSION THREAD

**Discussion by sgebr01** [deta/cloud-docs#228] [Help]
When trying to deploy, I keep getting an error saying 
```Failed to update dependencies. Internal server error.```
 What's the issue? It was able to deploy changes, it only messes up with the dependencies.

**Comment by abdelhai** [comment]
1. Could you retry now.
2. Could you share your `requirements.txt` file.

**Reply by sgebr01** [reply]
I tried now, it still didn't work. 
This is all that's in my requirements.txt
```
fastapi==0.78.0
Pillow==9.1.1
```

**Reply by aavshr** [reply]
hi @sgebr01 can you please provide your Micro's ID? 

Can you also please try:
1. `deta deploy purge-dependencies` 
2. `deta deploy`

**Reply by sgebr01** [reply]
Sure, the id is "7b3b7b0b-e105-4f52-949b-b0b1d195e2a0". I'll try that out and let you know.

**Reply by sgebr01** [reply]
I tried it, the same issue occured again.

**Reply by aavshr** [reply]
hi @sgebr01 there is also `rembg` in your installation request which is seemingly causing an issue. Can you please make sure your requirements file has only the two mentioned dependencies if you don't need `rembg`?

Regardless, you should not get an `internal server error` and sorry for this, we will investigate.

**Reply by sgebr01** [reply]
Removing that did fix it - thanks! I no longer needed that package.

**Reply by robertreden24** [reply]
> hi @sgebr01 there is also `rembg` in your installation request which is seemingly causing an issue. Can you please make sure your requirements file has only the two mentioned dependencies if you don't need `rembg`?
> 
> Regardless, you should not get an `internal server error` and sorry for this, we will investigate.

I am facing the same issue however without the said package in my requirements.txt file. Are there any other solutions?

**Reply by aavshr** [reply]
hi @robertreden24 sorry for this, can you please provide your `requirements.txt` file?

**Reply by robertreden24** [reply]
fastapi==0.78.0
keras_vggface==0.6
matplotlib==3.5.2
mtcnn==0.1.1
numpy==1.22.4
Pillow==9.1.1
pydantic==1.9.1
scipy==1.8.1

**Reply by aavshr** [reply]
@robertreden24 thank you for sharing, in your case too you should not be getting an `internal server error` and we will fix it as soon as possible. However, Micros will not be able to support these requirements at the moment.
